### PR TITLE
Add tests for survey launched.

### PR DESCRIPTION
**Pull request by rjweeks**
# Motivation and Context
As a Response Operations Manager I need to know when an EQ has been launched

# What has changed
New test for survey launched event.

# How to test?
Build Case Processor and Case API locally and run in Docker Dev.

# Links
https://trello.com/c/oZbmaF7e/1054-log-surveylaunched-event-3
ONSdigital/census-rm-case-processor#51
https://github.com/ONSdigital/census-rm-case-processor/pull/51
https://trello.com/c/oZbmaF7e/1054-log-surveylaunched-event-3

### CODE REVIEW

**Comment by rjweeks** [comment]
[Log SurveyLaunched event (3)](https://trello.com/c/oZbmaF7e/1054-log-surveylaunched-event-3)

File: acceptance_tests/features/steps/survey_launched.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    with RabbitContext(exchange=Config.RABBITMQ_EVENT_EXCHANGE) as rabbit:
+        rabbit.publish_message(
+            message=message,
+            content_type='application/json',
+            routing_key=Config.RABBITMQ_SURVEY_LAUNCHED_ROUTING_KEY)
+
+
+@step("the events logged for the survey launched case are {expected_event_list}")
+def check_refusal_event_logging(context, expected_event_list):
```
Comment (NickGrantX) [inline acceptance_tests/features/steps/survey_launched.py]: this code doesn't need to be duplicated... we already have a step that's configurable to look for events of any type

Comment (AdamHawtin) [reply acceptance_tests/features/steps/survey_launched.py]: I don't have an issue with tailoring steps to the scenario, the code underneath is still being reused but it lets these steps use more specific, descriptive variable names like `context.survey_launched_case_id`.

Comment (NickGrantX) [reply acceptance_tests/features/steps/survey_launched.py]: fine

**Review by NickGrantX — APPROVED** [review]
looks alright to me

File: acceptance_tests/features/steps/survey_launched.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+from behave import step
+
+from acceptance_tests.features.steps.event_log import check_if_event_list_is_exact_match
+from acceptance_tests.utilities.rabbit_context import RabbitContext
+from config import Config
+
+
+@step("a survey launched for a created case is received")
+def create_refusal(context):
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/survey_launched.py]: This shouldn't be named `create_refusal`

File: acceptance_tests/features/steps/survey_launched.py (hunk 1 above)
Comment (AdamHawtin) [reply acceptance_tests/features/steps/survey_launched.py]: This function should not be named `check_refusal_event_logging` though

Comment (AdamHawtin) [inline acceptance_tests/features/steps/survey_launched.py]: This function should not be named `check_refusal_event_logging`.

**Review by AdamHawtin — DISMISSED** [review]

**Comment by AdamHawtin** [comment]
Couple of small naming issues, otherwise the good to go!

File: acceptance_tests/features/steps/survey_launched.py (hunk 2 above)
Comment (rjweeks) [reply acceptance_tests/features/steps/survey_launched.py]: Changed.

File: acceptance_tests/features/steps/survey_launched.py (hunk 1 above)
Comment (rjweeks) [reply acceptance_tests/features/steps/survey_launched.py]: Changed.

**Review by ryangrundy7 — APPROVED** [review]
Tests all pass for me 👍

**Pull request merged**